libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -386,20 +401,17 @@ export interface IBucketClient {
    * @inflight
    */
   metadata(key: string): Promise<ObjectMetadata>;
-}
 
-/**
- * Metadata of a bucket object.
- */
-export interface ObjectMetadata {
-  /** The size of the object in bytes. */
-  readonly size: number;
-
-  /** The time the object was last modified. */
-  readonly lastModified: Datetime;
-
-  /** The content type of the object, if it is known. */
-  readonly contentType?: string;
+  /**
+   * Copy an object to a new location in the bucket. If the destination object
+   * already exists, it will be overwritten.
```

Comment:
```suggestion
  /**
   * Copy an object to a new location in the bucket. If the destination object
   * already exists, it will be overwritten. Returns once the copying is finished.
```